src/Admin/AdherentAdmin.php

```diff
@@ -423,10 +421,6 @@ protected function configureFormFields(FormMapper $formMapper)
                     ])
                 ->end()
                 ->with('Responsabilités locales', ['class' => 'col-md-6'])
-                    ->add('coordinatorCommitteeArea', CoordinatorManagedAreaType::class, [
-                        'label' => 'coordinator.label.codes.committee',
-                        'sector' => CoordinatorAreaSectors::COMMITTEE_SECTOR,
```

Comment:
`CoordinatorAreaSectors.php` can you remove this class too and adjust the Fixture class?